Src/RhinoPlugin.Test.Xunit/XunitTestInitFixture.cs

```diff
@@ -34,8 +34,8 @@ public XunitTestInitFixture()
             // Set path to rhino system directory
             string envPath = Environment.GetEnvironmentVariable("path");
             string programFiles = Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles);
-            systemDir = System.IO.Path.Combine(programFiles, "Rhino 7 WIP", "System");
-            systemDirOld = System.IO.Path.Combine(programFiles, "Rhino WIP", "System");
+            systemDir = System.IO.Path.Combine(programFiles, "Rhino 7", "System");
+            systemDirOld = System.IO.Path.Combine(programFiles, "Rhino 7 WIP", "System");
```

Comment:
Hi @paireks, thanks for the PR. Its been a while since I looked at this though. Is systemDirOld needed any more do you think?

Reply:
Hey! I'm not sure, maybe for some guys that still have WIP version, but this version is gonna expire soon if it's not yet. For me  systemDir is enough.